packages/repository-tests/src/crud/relations/acceptance/has-many.relation.acceptance.ts

```diff
@@ -0,0 +1,253 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect, toJSON} from '@loopback/testlab';
+import * as _ from 'lodash';
+import {
+  CrudFeatures,
+  CrudRepositoryCtor,
+  CrudTestContext,
+  DataSourceOptions,
+} from '../../..';
+import {
+  deleteAllModelsInDefaultDataSource,
+  withCrudCtx,
+} from '../../../helpers.repository-tests';
+import {Customer, Order} from '../fixtures/models';
+import {givenBoundCrudRepositories, mixedIdType} from '../helpers';
+
+export function hasManyRelationAcceptance(
+  dataSourceOptions: DataSourceOptions,
+  repositoryClass: CrudRepositoryCtor,
+  features: CrudFeatures,
+) {
+  describe('HasMany relation (acceptance)', () => {
+    before(deleteAllModelsInDefaultDataSource);
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let customerRepo: any;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let orderRepo: any;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let existingCustomerId: any;
```

Comment:
Can we use `unknown` instead of `any`?

Same comment applies to all other places using `any` for a variable storing an existing id.